app/views/courses/applications_open/_form_fields.html.erb

```diff
@@ -31,7 +31,7 @@
             What date will applications open?
           </legend>
           <span class="govuk-hint">
-            For example, 31 10 <%= @recruitment_cycle.year %>
+            For example, 30 09 <%= @recruitment_cycle.year %>
```

Comment:
Works without the padded zero so i think example should omit?:
```suggestion
            For example, 30 9 <%= @recruitment_cycle.year %>
```

Reply:
Thanks Phil